internal/cronsumer.go

```diff
@@ -62,6 +62,11 @@ func (k *kafkaCronsumer) Listen(ctx context.Context, strategyName string, cancel
 
 		msg := NewMessageWrapper(*m, strategyName)
 
+		if k.cfg.Consumer.HeaderFilterFn != nil && k.cfg.Consumer.HeaderFilterFn(msg.Headers) {
```

Comment:
We can also add headerFn in `kafkaCronsumer` struct like we did for consumeFn.

Reply:
We thought about it and it will filter only when consuming, isn't it better in consumer part of config?

Reply:
Reasonable :D

Reply:
I mean, instead of access headerFilterFn via k.cfg.Consumer (`k.cfg.Consumer.HeaderFilterFn(msg.Headers)`), we can extract a field named headerFilterFn within `kafkaCronsumer` struct so we can access like `k.headerFilterFn(msg.Headers)`

Reply:
I got it now :D, we can do that, so I think it still will be defined in config, but we are gonna pass it to `newKafkaCronsumer` function in `newCronsumer` function. Am i right?

Reply:
I don't understand :D I mean

![image](https://github.com/Trendyol/kafka-cronsumer/assets/19360039/befcc153-294e-467c-abd1-1e292a89362c)